### PR TITLE
Editing pass on Sections 1-5 of manual

### PR DESCRIPTION
**Pull request by prckent**
Edits from Charlie Horak (ORNL) for style and consistency plus bibliography fix.

### CODE REVIEW

File: manual/features.tex [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
+\item Wavefunction updates using optimized multideterminant algorithm of Clark et al.
 \item Backflow wavefunctions
 \item One, two, and three-body Jastrow factors
 \item Excited state calculations via flexible occupancy assignment of Slater determinants
-\item All electron and non-local pseudopotential calculations
-\item Casula T-moves for variational evaluation of non-local
-  pseudopotentials (non-size consistent and size-consistent variants)
+\item All electron and nonlocal pseudopotential calculations
+\item Casula T-moves for variational evaluation of nonlocal
+  pseudopotentials (nonsize consistent and size-consistent variants)
```
Comment (markdewing) [inline manual/features.tex]: I don't think 'nonsize' is a word, and 'size' should be more connected to 'consistent' the preceeding 'non'.   Should this be 'non-size-consistent'?

Comment (prckent) [reply manual/features.tex]: Fixed

File: manual/installation.tex [hunk 2]
```diff
@@ -1562,26 +1555,25 @@ \subsubsection{NiO performance tests}
...
-  \caption{System sizes and names for NiO performance tests. GPU performance
-    tests are named similarly but have different walker counts.}
+
   \label{tab:niotests}
 \end{center}
 \end{table}
 
 \subsection{Troubleshooting tests}
-ctest reports briefly pass or fail of tests in printout and also collects all the standard outputs to help investigating how tests fail.
+Ctest reports briefly pass or fail of tests in printout and also collects all the standard outputs to help investigating how tests fail.
```
Comment (markdewing) [inline manual/installation.tex]: I think CTest should be capitalized in the same style as CMake

File: manual/running.tex [hunk 3]
```diff
@@ -102,31 +102,31 @@ \subsection{Nested OpenMP threads}
...
 
 \subsection{Memory considerations}
-When using threads, some memory objects shared by all the threads. Usually these memory are read-only when the walkers are evolving, for instance the ionic distance table and wavefunction coefficients.
+When using threads, some memory objects are shared by all the threads. Usually these memory objects are read only when the walkers are evolving, for instance the ionic distance table and wavefunction coefficients.
```
Comment (markdewing) [inline manual/running.tex]: The second sentence should leave the hyphen in 'read-only'.   While 'read only' parses correctly, it does not convey the right intention - 'read-only' means 'not written', while 'read only' means 'only read during this phase'.

Comment (prckent) [reply manual/running.tex]: Fixed

File: manual/installation.tex (hunk 2 above)
Comment (prckent) [reply manual/installation.tex]: Fixed throughout

**Pull request merged**